app/helpers/status_tag_helper.rb

```diff
@@ -0,0 +1,13 @@
+module StatusTagHelper
+  def status_tag(active)
+    if active
+      "<span class='inline-flex px-2 text-xs font-semibold leading-5 text-green-800 bg-green-100 rounded-full'>
```

Comment:
interpolate only the css class that changes, you could use the `token_list/class_names` helper might come handy